getting-started/templates/systemlink-values.yaml

```diff
@@ -840,16 +840,7 @@ argoworkflows:
         registry: *imageRegistryRef
       ## Maximum number of workflows that can run in parallel.
       ##
-      parallelism: 300
-      workflowDefaults:
```

Comment:
These are identical values with what we set by default in SystemLink chart. Removed them since it's not a requirement to set them. I will also update https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/configuring-the-notebook-exectution-service.html to note that this configuration is optional.